Add Deprecation Notice to 5.7.0 (#5734)
TykTechnologies/tyk-docs: tyk-docs/content/product-stack/tyk-charts/tyk-control-plane-chart.md

{{< note success >}}
**Note** 

> SQLite support has reached end of life as of Tyk 5.7.0. To avoid disruption, please transition to [PostgreSQL]({{< ref"planning-for-production/database-settings/postgresql#introduction" >}}), [MongoDB]({{< ref "planning-for-production/database-settings/mongodb" >}}), or one of the listed compatible alternatives.

Reviewer: ```suggestion
Tyk no longer supports SQLite as of Tyk 5.7.0. To avoid disruption, please transition to [PostgreSQL]({{< ref"planning-for-production/database-settings/postgresql#introduction" >}}), [MongoDB]({{< ref "planning-for-production/database-settings/mongodb" >}}), or one of the listed compatible alternatives.
```
